teams/teams-ps/teams/New-CsTeamsEventsPolicy.md

```diff
@@ -0,0 +1,145 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/new-csteamseventspolicy
+schema: 2.0.0
+---
+
+# New-CsTeamsEventsPolicy
+
+## SYNOPSIS
+This cmdlet allows you to create a new TeamsEventsPolicy instance and set it's properties. Note that this policy is currently still in preview.
+
+## SYNTAX
+
+```
+New-CsTeamsEventsPolicy [-Identity] <String> [-AllowWebinars <String>] [-Description <String>]
+ [-EventAccessType <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+TeamsEventsPolicy is used to configure options for customizing Teams Events experiences.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> New-CsTeamsEventsPolicy -Identity DisablePublicWebinars -AllowWebinars Enabled -EventAccessType EveryoneInCompanyExcludingGuests
+```
+
+The command shown in Example 1 creates a new per-user Teams Events policy with the Identity DisablePublicWebinars. This policy disables a user from creating public webinars.
+
+### Example 2
+```powershell
+PS C:\> New-CsTeamsEventsPolicy -Identity DisableWebinars -AllowWebinars Disabled
+```
+
+The command shown in Example 2 creates a new per-user Teams Events policy with the Identity DisableWebinars. This policy disables a user from creating webinars.
+
+## PARAMETERS
+
+### -AllowWebinars
+This setting governs if a user can create Webinars using Teams Events. Set this to Disabled to disable a user from creating webinars
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EventAccessType
+This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
+Set this to EveryoneInCompanyExcludingGuests to disable a user from creating public webinars

```

Comment:
What are all the possible values of this parameter? Everyone and EveryoneInCompanyExcludingGuests? please specify